### PR TITLE
Remove duplicate Felix Health Port for Openshift

### PR DESCRIPTION
**Pull request by rene-dekker**
Remove duplicate Felix Health Port for Openshift

Our tests already were testing for the right port to be set.... twice. So no additional tests are needed.

### CODE REVIEW

**Review by caseydavenport — APPROVED** [review]

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Pull request merged**
